frontend/fingertips-frontend/components/organisms/ViewsWrapper/hasSufficientHealthDataCheck.ts

```diff
@@ -1,34 +1,43 @@
 import { IndicatorWithHealthDataForArea } from '@/generated-sources/ft-api-client';
 
-const hasAreaHealthData = (
+const hasSufficientAreaHealthData = (
   indicatorDataForArea: IndicatorWithHealthDataForArea,
-  areasSelected: string[]
+  areasSelected: string[],
+  indicatorsRequested: number
 ): boolean => {
   if (indicatorDataForArea.areaHealthData) {
     return indicatorDataForArea.areaHealthData.reduce<boolean>(
-      (hasHealthData, healthData) => {
+      (hasSufficientAreaHealthData, areaHealthData) => {
         const hasAreaSelectedHealthData = areasSelected?.find(
-          (areaSelected) => areaSelected === healthData.areaCode
+          (areaSelected) => areaSelected === areaHealthData.areaCode
         );
 
         if (hasAreaSelectedHealthData) {
-          return true;
+          const hasSufficientAreaHealthData =
+            indicatorsRequested === 1 && areasSelected.length <= 2
+              ? areaHealthData.healthData.length > 1
+              : true;
+          if (hasSufficientAreaHealthData) return true;
```

Comment:
Do we need to check the status of hasSufficentAreaHealthData? - we could just return that value or skip the line as it will be returned on line 22.

Reply:
We want to only overwrite the state of `hasSufficientAreaHealthData` when we have sufficient health data for the area. If we remove this check we overwrite the state and this test then fails;
`should return true if the indicator has sufficient health data for at least one of the areaSelected`
The test here has 2 areas with the last one having insufficient health data. We then incorrectly set the entire state to false when it should be true since the first area had sufficient health data.